roles/customer_support_representative.md

```diff
@@ -0,0 +1,93 @@
+# Customer Support Representative
+
+Our Business Development Reps are at the forefront of building relationships with markets, organisations and key stakeholders. Ability to execute and evolve a plan is key.
+
+## What does the job entail?
+
+Made Tech are growing and winning more new projects with both existing and new customers. We have a reliability engineering team responsible for running live services for our customers and delivering a roadmap of improvement for applications and infrastructure including new software features, automation, bug fixes and support issues. Due to our customer base expanding and the number of projects growing, we’re experiencing an increase in technical support enquiries.
+
+We need a dedicated, single point of contact to deal quickly and effectively with our customer support issues in an empathetic and customer focused way, triaging and escalating queries to ensure effective use of time and swift resolution.
+
+This role will report to our Delivery and Operations Director and liaise directly with both customers and reliability engineers, some of whom work remotely.
+
+Day-to-day you will receive incoming requests such as outages, error messages and performance issues from a variety of sources including Slack, email, video calls on Google Hangouts, etc. You will triage the enquiry to either escalate accordingly (e.g. high priority systems outages) or add to our backlog (low priority e.g. small bug fixes or systems performance issues), responding to all customer queries using ZenDesk to manage email queries and triage.
+
+You will escalate high priority issues to our reliability engineers and add low priority requests to the roadmap to be addressed at a later date. We use Asana to stay organised, so you will add each request to our customer boards as you receive them.
+
+You will act as single point of contact to ensure customers are informed and updated on the status of their technical support enquiry. You will also lead fortnightly reviews with each customer, discussing what has been delivered in previous iterations and which priorities remain for the following two weeks.
+
+Over time you will form detailed knowledge of the various applications we have created for customers, including their purpose, functionality, hosting, metrics and permissions. This is information you will use to make support more effective, resolving issues yourself if needed, and continuing to ensure ever more effective triaging where not.
```

Comment:
I think there's something maybe even bigger in here about understanding customer goals, where we can help further our impact with their organisation etc.?